rotkehlchen/tests/unit/accounting/test_basic.py

```diff
@@ -81,41 +84,31 @@ def test_simple_accounting(accountant, google_service, expected_pnls):
         }),
     ),
 ])
+@pytest.mark.parametrize('accounting_initialize_parameters', [True])
 def test_selling_crypto_bought_with_crypto(accountant, google_service, expected_pnl_totals):
     history = [
-        Trade(
-            timestamp=Timestamp(1446979735),
+        *create_swap_events(
+            timestamp=TimestampMS(1446979735000),
             location=Location.EXTERNAL,
-            base_asset=A_BTC,
-            quote_asset=A_EUR,
-            trade_type=TradeType.BUY,
-            amount=FVal(82),
-            rate=Price(FVal('268.678317859')),
-            fee=None,
-            fee_currency=None,
-            link=None,
-        ), Trade(
-            timestamp=Timestamp(1449809536),  # cryptocompare hourly BTC/EUR price: 386.175
+            event_identifier=uuid4().hex,
```

Comment:
my point with those is that you can make them a constant and avoid function calls. I see that you do it a lot in the code. Is there any good reason to not have just a string here?